### ISSUE TITLE
Errors out when start up local server

### ISSUE THREAD

**Issue by peco8** [dry-rb/dry-validation#303]
I have errors rendered below whenever start the rails server locally? 

```
/Users/test/Desktop/billing/vendor/bundle/ruby/2.3.0/bundler/gems/dry-validation-46e44d1d1bf5/lib/dry/validation/message_compiler.rb:13: warning: already initialized constant Dry::Validation::MessageCompiler::EMPTY_OPTS

/Users/test/Desktop/billing/vendor/bundle/ruby/2.3.0/gems/dry-core-0.2.3/lib/dry/core/constants.rb:52: warning: previous definition of EMPTY_OPTS was here

```

Is this a bug?

**Comment by solnic** [comment]
We moved these consts to dry-core that's why there's a warning. I thought we fixed that in master though. I'll deal with it and release a new version later this week.

**Comment by solnic** [comment]
Oh wait, it looks like it was fixed in 0.10.4, see [here](https://github.com/dry-rb/dry-validation/blob/master/CHANGELOG.md#v0104-2016-12-03). Could you check 0.10.4?

**Comment by peco8** [comment]
Oh yea It's all fixed in `0.10.4`.  Could you create the tag for that latest version?
I could do `gem 'dry-validation', git: 'git://github.com/dry-rb/dry-validation.git', tag: 'v0.10.4'`

**Comment by solnic** [comment]
Ooops, will do
